<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdvPrintCaptionPage</class>
 <widget class="QWidget" name="AdvPrintCaptionPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>920</width>
    <height>487</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string/>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_12">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_9">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Captions Type:</string>
              </property>
              <property name="buddy">
               <cstring>m_captions</cstring>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="m_captions">
              <item>
               <property name="text">
                <string>No captions</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Image file names</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Exif date-time</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Comments</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Free</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QLineEdit" name="m_FreeCaptionFormat">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="minimumSize">
               <size>
                <width>400</width>
                <height>0</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QGridLayout" name="gridLayout_3">
              <item row="0" column="0">
               <widget class="QLabel" name="m_free_label1">
                <property name="text">
                 <string>&quot;%f&quot; : file name
&quot;%t&quot; : camera exposure time
&quot;%c&quot; : item comment
&quot;%i&quot; : ISO sensibility from camera
&quot;%d&quot; : date and time from camera
</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QLabel" name="m_free_label2">
                <property name="text">
                 <string>&quot;%r&quot; : camera sensor resolution
&quot;%a&quot; : camera aperture
&quot;%l&quot; : focal length from camera
&quot;\n&quot; : new line</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_6">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_6">
                <item>
                 <widget class="QLabel" name="label_4">
                  <property name="text">
                   <string>Color:</string>
                  </property>
                  <property name="buddy">
                   <cstring>m_font_color</cstring>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="Digikam::DColorSelector" name="m_font_color"/>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_7">
                <item>
                 <widget class="QLabel" name="textLabel1_2">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="text">
                   <string>Size:</string>
                  </property>
                  <property name="wordWrap">
                   <bool>false</bool>
                  </property>
                  <property name="buddy">
                   <cstring>m_font_size</cstring>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QSlider" name="m_font_size">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="minimum">
                   <number>2</number>
                  </property>
                  <property name="maximum">
                   <number>8</number>
                  </property>
                  <property name="singleStep">
                   <number>2</number>
                  </property>
                  <property name="pageStep">
                   <number>2</number>
                  </property>
                  <property name="value">
                   <number>4</number>
                  </property>
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_5">
              <item>
               <widget class="QLabel" name="label_3">
                <property name="text">
                 <string>Family:</string>
                </property>
                <property name="buddy">
                 <cstring>m_font_name</cstring>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QFontComboBox" name="m_font_name"/>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <item>
               <widget class="QCheckBox" name="m_sameCaption">
                <property name="text">
                 <string>Same to all</string>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="m_setDefault">
                <property name="text">
                 <string>Use these settings as default</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Digikam::DColorSelector</class>
   <extends>QPushButton</extends>
   <header>dcolorselector.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
